app/public/css/styles.css

```diff
@@ -1270,6 +1260,13 @@ button[data-balloon] {
   max-height: 178px;
 }
 
+.authentication-image-container {
+  position: relative;
+  padding-bottom: calc(178px + 24px); 
```

Comment:
What is the purpose of this? From what I know, calc is used mainly for percentages combined with pixel values.

Reply:
This was just to emphasize that the constant comes from the addition of the image height and its margin. Moving this information to a comment.